apps/editors/tests/test_views.py

```diff
@@ -2733,6 +2733,13 @@ def test_form_errors(self):
         doc = pq(r.content)
         eq_(doc('#editor-motd .errorlist').text(), 'This field is required.')
 
+    def test_motd_tab(self):
+        self.login_as_admin()
+        r = self.client.get(self.get_url())
+        announcement_tab = pq(r.content)(
+            'li.top:nth-child(5) > a:nth-child(1)').text()
```

Comment:
this test might break when the layout gets shifted around. A better approach would be to add an id to the Announcement anchor and just check that it's visible. But this test is good for now, we can always fix it up later if it becomes a burden.